src/Codeception/Lib/Connector/ZF2.php

```diff
@@ -180,4 +175,33 @@ private function createApplication()
             $events->detach([$sendResponseListener, 'sendResponse']); //ZF3
         }
     }
+
+    private function bootstrapApplication($configuration = [])
```

Comment:
Please add comment that this method is copy of `Zend\Application::init` method with specific changes to support persisted services,